average.js

```diff
@@ -1,6 +1,23 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  
+  let result = 0;
+  let countNan = 0;
+  
+  for (let i = 0; i<numbers.length; i++)
```

Comment:
This code will no doubt satisfy the requirement :)
JavaScript has more expressive constructs like `forEach` and `map`.
Try using them to reduce extra indexes like `i`

Reply:
`
function average(numbers) {
    
    let result = 0;
    let countNan = 0;
    
    numbers.forEach((number) => isNaN(number) ? countNan+=1 : result+=number);
    return result / (numbers.length - countNan)
}

module.exports = {average};
`

Reply:
Implement the function using forEach constructs to reduce extra indices.